jobs/test_utils.py

```diff
@@ -19,16 +19,23 @@
 from __future__ import absolute_import  # pylint: disable=import-only-modules
 from __future__ import unicode_literals  # pylint: disable=import-only-modules
 
-from core.tests import test_utils
+import datetime
+
+from core.tests import test_utils as core_test_utils
```

Comment:
Can we just keep the existing filename as jobs_test_utils (or maybe make it more specific, like jobs_test_classes)? We talk verbally about test_utils quite often; it would be better to not have the potential for collision.

Reply:
Done.

Reply:
Done.